README.md

```diff
@@ -17,3 +18,48 @@ In order to run the MIMIC-IV ETL, simply run the following command:
 In order to run the MIMIC-IV ETL, simply run the following command:
 
 `meds_etl_omop omop omop-esds` where omop is a folder containing csv files (optionally gzipped) for an OMOP dataset. Each table should either be a csv file with the table name (such as person.csv) or a folder with the table name containing csv files.
+
+## MEDS CSV
+
+The MEDS schema can be a bit tricky to directly ETL into as it is a nested parquet schema and nested schemas are not supported by many pieces of software. In additional, the global join of combining all of a patient's data into a single value can be difficult to implement.
+
+In order to make things simpler for users, we have a special MEDS CSV schema, and ETLs back and forth between MEDS and MEDS CSV.
+
+This MEDS CSV schema is 1-1 compatible with MEDS. It is a simple tabular schema that has four core columns: patient_id, time, code, value. These columns correspond to the core ESDS columns, with the one exception that all the value columns have been combined into a single one.
+
+patient_id is a 64-bit integer, time is an ISO 8601 datetime, and code is a string. value is either a number, an ISO 8601 datetime, or arbitrary text. 
+
+These columns get converted into the standard ESDS columns, with value converted to either numeric_value, datetime_value or text_value as necessary. 
+
+The simple CSV format also supports arbitrary additional columns, that get converted into a metadata struct.
+
+For example, the following CSV would be converted into the following MEDS patient:
+
+Input CSV:
+```
+patient_id,time,code,value,clarity_source
```

Comment:
@EthanSteinberg `clarity_source` is just a generic example metadata column, right, not something you're expecting to be universally present?

Reply:
Correct. Renamed to arbitrary_metadata_column to be more explicit.